src/site/content/en/fast/optimize-css-background-images-with-media-queries/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: Optimize CSS background images with media queries
+authors:
+  - demianrenzulli
+description: |
+  Use media queries to send images that are only as large as they need to be, 
+  a technique commonly known as responsive images.
+date: 2020-03-05
+updated: 2020-03-05
+---
+
+Many sites request heavy resources, like images, that are not optimized for certain screens, and send large CSS files containing styles that some devices will never use. Media queries are a popular technique for delivering tailored stylesheets and assets to different screens to reduce the amount of data transferred to users and improve page load performance. This guide shows you how to use media queries to send images that are only as large as they need to be, a technique commonly known as **responsive images**.
```

Comment:
Minor nit: it's not really accurate to describe media queries as a technique. Media queries are a feature, but using media queries for this purpose could be considered a technique.

How about changing it to start: "Using media queries is a popular technique..."

Reply:
```suggestion
Many sites request heavy resources, like images, that are not optimized for certain screens, and send large CSS files containing styles that some devices will never use. Using media queries is a popular technique for delivering tailored stylesheets and assets to different screens to reduce the amount of data transferred to users and improve page load performance. This guide shows you how to use media queries to send images that are only as large as they need to be, a technique commonly known as **responsive images**.
```